src/impl/tpls/KokkosBlas_Host_tpl.cpp

```diff
@@ -249,6 +249,23 @@ extern "C" {
                                      const std::complex<double>*,
                                      const std::complex<double>*, int*,
                                      /* */ std::complex<double>*, int* );
+
+  ///
+  /// Gesv
+  ///
+
+  void F77_BLAS_MANGLE(sgesv,SGESV)( int*, int*,
```

Comment:
These are not BLAS functions, they are LAPACK functions.  They may live in a different library.  It's plausible that people will want to build and run on platforms that don't have a Fortran compiler, and thus can't build LAPACK.  Thus, I think it would make sense to separate out LAPACK `extern "C"` declarations into a different `.cpp` file.

Reply:
Thanks, @mhoemmen . See my comment below.